.codecov.yml

```diff
@@ -18,7 +18,9 @@ coverage:
     # Learn more at https://docs.codecov.io/docs/commit-status
     project: true
     patch: true
-    changes: false
+    changes: true
+    # Avoids showing a red cross for most PRs
```

Comment:
👍

Reply:
Well that's the theory. I'm not sure whether it will actually work as expected ...